code/game/objects/effects/decals/posters/asters.dm

```diff
@@ -0,0 +1,58 @@
+// Asters Guild faction posters
+/datum/poster/asters/asterite
+	name = "Voyage of The Asterites"
+	desc = {"A flourished bit of symbolism representing the first Asterite's great exodus from Humanity's origin, now posed to to inherit the stars themselves. \
+	At least, that's what the small text says."}
```

Comment:
I don't think Asterite is a thing.
Same as "NeoTheologian" god forgive us all

Reply:
> Not sure about the meaning of it, Aster Guild is a corporation that is 60 or so years old, is that corporate attempt on rewriting history for some reason and posing as nation dating back to Exodus from Earth? If so, why?

This is essentially it, the Asters Guild positioning themselves in events which simply aren't true. This, of course, can get confusing for the backend of keeping actual "continuity". "Asterite" is meant refer to the actual members of the Guild, such as the three aboard our ship, the Captain, First Officer, and Guild Merchant. They're "Asterites", their workers are just "Guildies". Issue is the term isn't really widespread in use.

Reply:
This is also probably the poster I'm least satisfied with, and will probably re-write to something more straight forward and inoffensive. Since I agree the poster seems out-of-character.

Reply:
Frankly, "Asterite" and "Guildie" sounds really off, there might be better alternative.